action.html

```diff
@@ -275,6 +268,19 @@ <h4 id="graphics">And some of our most popular <a href="https://tbfighters.notio
 						</noscript>
 					</div>
 				</div>
+				
+				<div class="wrapper-full" style="transition: ease 0.5s all;">
```

Comment:
the `style="transition: ease 0.5s all;"` doesn't need to be here

Reply:
Thank you!!